core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -191,12 +191,13 @@ object TopicCommand extends Logging {
     val topics = getTopics(zkUtils, opts)
     val reportUnderReplicatedPartitions = if (opts.options.has(opts.reportUnderReplicatedPartitionsOpt)) true else false
     val reportUnavailablePartitions = if (opts.options.has(opts.reportUnavailablePartitionsOpt)) true else false
+    val reportUnbalancedPartitions = if (opts.options.has(opts.reportUnbalancedPartitionsOpt)) true else false
     val reportOverriddenConfigs = if (opts.options.has(opts.topicsWithOverridesOpt)) true else false
```

Comment:
@ijuma I think these were added to avoid using same `opts.options.has...` at bunch of places in this class.